README.md

```diff
@@ -1,27 +1,39 @@
-# dojot Deploy - Docker compose
+# Dojot Deploy - Docker Compose
 
-__Attention__: Before running this deployment, it is necessary to define your domain or IP in the [.env](./.env) file in the variable ``DOJOT_DOMAIN_NAME``.
+This deployment option is best suited to development and functional environments. For production environment we recommend to use [Kubernetes](https://github.com/dojot/ansible-dojot).
 
-__Note__ To get completely ready, **healthy**, all services in this `docker-compose` take an average of at least 12 minutes.
+## Getting Started
 
 This repository contains the necessary configuration files
-for quick deployment of the dojot platform using docker-compose.
+for quick deployment of the Dojot Platform using `docker-compose`.
 
-For instructions on how to get it up and running, please check [Installation Guide](https://dojotdocs.readthedocs.io/en/latest/installation-guide.html#docker-compose). **Always change the ``admin`` user password to a suitable password and keep it safe.**
+### Requirements
```

Comment:
Sugiro manter:

For instructions on how to get it up and running, please check [Installation Guide](https://dojotdocs.readthedocs.io/en/latest/installation-guide.html#docker-compose). **Always change the ``admin`` user password to a suitable password and keep it safe.**

Reply:
ok.. informação disponível na linha 32